assets/js/theme/common/faceted-search.js

```diff
@@ -408,6 +410,18 @@ class FacetedSearch {
             this.collapsedFacets = _.without(this.collapsedFacets, id);
         }
     }
+
+    onPopState() {
+        const currentUrl = window.location.href;
+        const searchParams = new URLSearchParams(currentUrl);
+        // If searchParams does not contain a page value then modify url query string to have page=1
+        if (!searchParams.has('page')) {
+            const linkUrl = $('.pagination-link').attr('href');
+            const url = `${linkUrl.slice(0, linkUrl.indexOf('page'))}page=1`;
```

Comment:
Why does `linkUrl.indexOf('page')` return something if `searchParams.has('page')` is false?

Reply:
linkUrl is taken from the dom searchParams is taken from the url.  Each page has a corresponding href on the dom.

Reply:
I think we need to use something more robust that updates just the page parameter. `linkUrl` could contain other search parameters when facets are selected, and this code assumes that `page` is the last parameter, which may not always be the case.